GVRf/Sample/scene-objects/res/layout/main.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
License header misses(similar issues also in some other xml files).

And also, I don't think for this sample app the layout xml is used, maybe you can remove it for clean up purpose.